.ado/variables/github.yml

```diff
@@ -0,0 +1,6 @@
+variables:
+  # react-native-platform-override calls GitHub's API, and may be rate-limited
+  # if unauthenticated. This is run during PRs, so it it unsafe to use a token
+  # with private permissions. The below PAT has access to only public information
+  Platform.Override.Github.Token: 9006d7c9a9dda0a33d475aefba41ebfee2169396
```

Comment:
Still need to check with folks as to whether there are any issues doing this. It feels a bit sketchy still, even if the token has zero permissions.

Reply:
Had a conversation with some folks to try to find a good solution here. Current thinking is to have an AZP variable group for public tokens used in the PR pipeline that are zero permission and well-documented.